src/uniprotkb/components/entry/tabs/genomic-coordinates/Entries.tsx

```diff
@@ -0,0 +1,346 @@
+        </Link>
+      </h4>
+      <InfoList infoData={infoData} columns />
+    </section>
+  );
+};
+
+type EntriesProps = {
+  entries: Record<string, Array<FlatGenomicEntry>>;
+  index: number;
+  isoformIDs: string[];
+};
+
+const Entries = ({ entries, index, isoformIDs }: EntriesProps) => {
+  const accessionEntriesPairs = Object.entries(entries);
+  // First pair, entry info, first flattened entry info
+  const representativeEntry = accessionEntriesPairs[0][1][0];
+
+  const { Ensembl } = useDatabaseInfoMaps()?.databaseToDatabaseInfo || {};
+
+  // Mostly from Ensembl
+  let xrefInfo: DatabaseInfoPoint | null = null;
+  if (representativeEntry.gnCoordinate.ensemblGeneId?.startsWith('ENS')) {
+    xrefInfo = Ensembl;
+  }
+
+  const infoData = [
+    {
+      title: 'Chromosome',
+      content: representativeEntry.gnCoordinate.genomicLocation.chromosome,
+    },
+    {
+      title: 'Strand',
+      content: representativeEntry.gnCoordinate.genomicLocation.reverseStrand
+        ? 'Reverse'
+        : 'Forward',
+    },
+    {
+      title: `${xrefInfo ? 'Ensembl g' : 'G'}ene ID`,
+      content:
+        representativeEntry.gnCoordinate.ensemblGeneId &&
+        (xrefInfo?.uriLink ? (
+          <ExternalLink
+            url={processUrlTemplate(xrefInfo.uriLink, {
+              id: representativeEntry.gnCoordinate.ensemblGeneId,
+            })}
+          >
+            {representativeEntry.gnCoordinate.ensemblGeneId}
+          </ExternalLink>
+        ) : (
+          representativeEntry.gnCoordinate.ensemblGeneId
+        )),
+    },
+  ];
+
+  const groupedExons = groupBy(
+    accessionEntriesPairs
+      .map(
+        ([, entries]) =>
+          entries.map((entry) =>
+            entry.gnCoordinate.genomicLocation.exon.map((exon) => ({
+              ...exon,
+              // Add accession info to each exon info before flattening
+              accession: entry.accession,
+            }))
+          ),
+        2
+      )
+      .flat(2)
+      .sort(
+        sortExons(
+          representativeEntry.gnCoordinate.genomicLocation.reverseStrand
+        )
+      ),
+    (data) => data.id
+  );
+
+  const [mappedIsoforms, notMappedIsoforms] = partition(
+    isoformIDs,
+    (isoformID) =>
+      accessionEntriesPairs.some(([accession]) => accession === isoformID)
+  );
+
+  return (
+    <Card
+      header={
+        <h3>
+          Genomic location{' '}
+          {representativeEntry.gnCoordinate.ensemblGeneId
+            ? // TODO: better gene name whenever the endpoint provides it
+              `for gene ${representativeEntry.gnCoordinate.ensemblGeneId}`
+            : index}
+        </h3>
+      }
+    >
+      <InfoList infoData={infoData} columns />
+      {accessionEntriesPairs.map(([accession, entries]) => (
+        <Entry key={accession} entries={entries} xrefInfo={xrefInfo} />
+      ))}
+      <DatatableWrapper alwaysExpanded>
+        <table className={helper['no-wrap']}>
```

Comment:
Is it possible to increase the column padding to help distinguish the content. It's a bit tricky in the extreme cases to know where one column starts and finishes:

<img width="533" alt="Screenshot 2023-10-20 at 12 24 43" src="https://github.com/ebi-uniprot/uniprot-website/assets/4357962/ef9495aa-0626-48fe-bfac-5e1b2d461d19">

Reply:
I've added an inline padding of 1ch on each side of each cell, so there should be a bit more breathing room now